### PR TITLE
Polyhedron demo: fix the display of axis

### PR DESCRIPTION
**Pull request by maxGimeno**
- The axis system was representing the rotation of the camera, which was not
  very clear. It now represents the mvp matrix, without the translation part
  which is less disturbing.

### CODE REVIEW

File: Polyhedron/demo/Polyhedron/Viewer.cpp [hunk 1]
```diff
@@ -158,13 +158,9 @@ void Viewer::initializeGL()
       "   fP = mv_matrix * vertex; \n"
       "   fN = mat3(mv_matrix)* normal; \n"
       "   vec4 temp = vec4(mvp_matrix * vertex); \n"
-      "   vec4 ort = ortho_mat * vec4(width, height, 0,0); \n"
+      "   vec4 ort = ortho_mat * vec4(width-150, height-150, 0,0); \n"
```
Comment (sloriot) [inline Polyhedron/demo/Polyhedron/Viewer.cpp]: What is this magic 150 number?

Comment (lrineau) [inline Polyhedron/demo/Polyhedron/Viewer.cpp]: Probably the width of the display of axis, in pixels.

File: Polyhedron/demo/Polyhedron/Viewer.cpp [hunk 2]
```diff
@@ -726,10 +727,8 @@ void Viewer::resizeGL(int w, int h)
     {
         orthoMatrix.data()[i] = (float)ortho[i];
     }
-    int max = w;
-    if (h>w)
-        max = h;
-    QVector4D length(max,max,max, 1.0);
+
+    QVector4D length(60,60,60, 1.0);
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/Viewer.cpp]: Here is another hard-coded constant: 60. Probably related to the length of the arrows, in pixels.

**Comment by lrineau** [comment]
Will be tested in [CGAL-4.8-Ic-19][t].

[t]: https://cgal.geometryfactory.com/CGAL/Members/testsuite/results-4.8-Ic-19.shtml

**Comment by maxGimeno** [comment]
I didn't see any error/warning that could have been caused by this PR, I think it can be merged.

**Pull request merged**